extensions/escape-game/standalone.mjs

```diff
@@ -957,27 +953,23 @@ class Team {
     return true;
   }
 
-  /**
-   * Start all containers
-   */
-  async startContainers() {
-    return await this.containerLocks.acquire('containers', async () => {
-      for (const defaultTerminal of this.defaultTerminals) {
-        try {
-          let ret = await promisify(this.terminalServerGrpcService.CreateContainer.bind(this.terminalServerGrpcService))({
-            imageName: defaultTerminal.imageName
-          }, {deadline: new Date(Date.now() + 20000)});
-
-          if (!ret.success) {
-            console.error(`Fail to start container with image ${defaultTerminal.imageName}.`);
-            return false;
-          }
-          this.terminals.set(defaultTerminal.terminalId, ret.containerId);
-        } catch (e) {
-          console.error('Failed to start container: ', e);
-        }
+  async ensureContainerAvailable(terminalId) {
```

Comment:
Please add jsdoc.
Also please comment that all containers will be reaped by default if unused for a certain time. This behaviour is fixed and cannot be changed.